packages/mobile/src/utils/translationsFetcher.ts

```diff
@@ -0,0 +1,11 @@
+import i18n from 'src/i18n'
+
+export const fetchI18nContent = (content: { [lang: string]: any }) => {
```

Comment:
nit: fetch often means a network fetch
```suggestion
export const getContentForCurrentLang = (content: { [lang: string]: any }) => {
```